inc/languages/english/admin/myalerts.lang.php

```diff
@@ -8,6 +8,8 @@
 $l['setting_myalerts_perpage_desc'] = "How many alerts do you wish to display on the alerts listing page? (default is 10)";
 $l['setting_myalerts_dropdown_limit'] = "Number of alerts to show in dropdown";
 $l['setting_myalerts_dropdown_limit_desc'] = "How many alerts do you wish to display in the global alerts dropdown? (default is 5)";
+$l['setting_myalerts_headericon'] = "Alert notification visibility mode (header)";
```

Comment:
This wording is unclear and needs improvement. One possibility: "Hide Alerts (0) header". There might be better ones.